app/services/permission.rb

```diff
@@ -32,8 +34,17 @@ def guest_user_permissions
 
   def registered_user_permissions
     return true if controller == "home"
-    return true if controller == "sessions"
-    return true if controller == "users"
+    return true if controller == "sessions" && action.in?(["destroy"])
+    return true if controller == "users" && action.in?(["show"])
+    return true if controller == "projects" && action.in?(["index", "show", "new", "create"])
+    return true if controller == "comments" && action.in?(["create"])
+    return true if controller == "user_funded_projects" && action.in?(["new"])
+  end
+
+  def project_owner_permissions
```

Comment:
Yes. But we don't currently have that functionality. When we add that functionality, we'll have to add that permission. @cews7